static/js/drafts.js

```diff
@@ -380,8 +382,38 @@ function remove_draft($draft_row) {
     if ($("#drafts_table .draft-row").length === 0) {
         $("#drafts_table .no-drafts").show();
     }
+    if ($("#drafts-from-conversation .draft-row").length === 0) {
+        $("#drafts-from-conversation .no-conversation-drafts").show();
+    }
+    if ($("#other-drafts .draft-row").length === 0) {
+        $("#other-drafts .no-other-drafts").show();
+    }
+    if (
+        $("#drafts-from-conversation .draft-row").length === 0 &&
+        $("#other-drafts .draft-row").length === 0
+    ) {
+        $("#split-drafts").hide();
+    }
```

Comment:
I'm not used to doing this sort of thing without a reactive framework, so I'd love feedback on if these parts seem reasonable.